src/components/Api.js

```diff
@@ -0,0 +1,153 @@
+export default class Api {
+  constructor({ baseUrl, headers }) {
+    this._baseUrl = baseUrl;
+    this._headers = headers;
+    // constructor body
+  }
+
+  async getInitialCards() {
+    try {
+      const res = await fetch(`${this._baseUrl}/cards`, {
+        method: "GET",
+        headers: this._headers,
+      });
+      if (!res.ok) {
+        throw new Error(`HTTP error! Status: ${res.status}`);
+      }
+      return await res.json();
+    } catch (err) {
+      console.log(err);
+    }
+  }
+
+  async getUserInfo() {
+    try {
+      const res = await fetch(`${this._baseUrl}/users/me`, {
+        method: "GET",
+        headers: this._headers,
+      });
+      return await res.json();
```

Comment:
`      return await res.json();
`

You need to check every response if it's `ok` or not according to the task